### PR TITLE
Don't try to submit again if using formplayer

### PR DESCRIPTION
**Pull request by wpride**
If using Formplayer, the server submits the XML to HQ rather than the browser so don't try to submit again from the browser. 

cross https://github.com/dimagi/formplayer/pull/76/files
cross https://github.com/dimagi/touchforms/pull/247

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
     $.fn.inlineTouchform = function (options) {
         var $target = $($(this).get(0));
-        options.onerror = options.onerror || function () {};
-        options.onload = options.onload || function () {};
+        options.onerror = options.onerror || function () {
+            };
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js]: Expected indentation of 8 space characters but found 12. (indent)

File: corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js [hunk 2]
```diff
@@ -11,8 +11,10 @@
...
+        options.onload = options.onload || function () {
+            };
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js]: Expected indentation of 8 space characters but found 12. (indent)

File: corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js [hunk 3]
```diff
@@ -32,16 +34,21 @@
...
+                        } else {
+                            // post to receiver
+                            $.ajax({
+                                type: 'POST',
+                                url: options.submitUrl,
+                                data: xml.output,
+                                success: function () {
+                                    $target.html(alertHtml('Form successfully submitted!', 'alert-success'));
+                                    options.onsubmit();
+                                }
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js]: Missing trailing comma. (comma-dangle)

**Comment by benrudolph** [comment]
👍 on dimagimon

File: corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js [hunk 4]
```diff
@@ -51,12 +56,13 @@
...
                     onload: function (adapter, resp) {
                         options.onload();
-                    }
+                    },
                 });
                 data.onLoading = onLoading;
                 data.onLoadingComplete = onLoadingComplete;
                 data.uses_sql_backend = options.uses_sql_backend;
                 data.formplayerEnabled = options.formplayerEnabled;
+                data.post_url = options.submitUrl;
```
Comment (czue) [inline corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js]: is there a cross for this part?

Comment (czue) [inline corehq/apps/cloudcare/static/cloudcare/js/touchforms-inline.js]: sorry i can't read. ignore

**Comment by czue** [comment]
any rhyme or reason to when you camelCase versus python_case?

**Comment by wpride** [comment]
@czue none, usually. In this case I think submitUrl was inherited from the javascript code and post_url from the extant touchforms code. I would be down with taking a pass to convert all params to one or the other though

**Comment by benrudolph** [comment]
The general strategy I try to follow is that JS is always camel when possible. If the attribute is derived from Python (like we wrap some attributes sent from the server) then keep the underscores

**Pull request merged**